.github/workflows/solvcon_install_devenv.yml

```diff
@@ -1,90 +1,45 @@
-name: solvcon_install_devenv
-
+name: Build with devenv
 on:
   schedule:
     - cron: '21 18 * * 5'
-
```

Comment:
Removing the line to align with the naming convention of `devenv` style so make us easier to maintain the CI. Let me know if anyone is against it. We can discuss and choose the other naming convention to fit SOLVCON project better.

Reply:
Look good to me.